### PR TITLE
Hbase

### PR DESCRIPTION
**Pull request by darmac**

### CODE REVIEW

File: var/spack/repos/builtin/packages/hbase/package.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    data storage provided by the Google File System, HBase provides
+    Bigtable-like capabilities on top of Apache Hadoop.
+    """
+
+    homepage = "https://github.com/apache/hbase"
+    url      = "https://github.com/apache/hbase/archive/rel/1.4.12.tar.gz"
+
+    version('2.2.2RC0', sha256='fa78e642225ee13c5b5f9e53f7eb3ec67c0188a776f5525d4d72362a40fcf653')
+    version('2.2.2',    sha256='e9a58946e9adff1cac23a0b261ecf32da32f8d2ced0706af1d04e8a67d582926', preferred=True)
+    version('2.1.8RC0', sha256='e2de965f2dcea83cbd96159d40c2ba9e4a9d0f459d70bc133408886954df627c')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/hbase/package.py]: No need to add the release candidates if the stable version has already been released.

File: var/spack/repos/builtin/packages/hbase/package.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    """
+    Apache HBase is an open-source, distributed, versioned, column-oriented
+    store modeled after Google' Bigtable: A Distributed Storage System for
+    Structured Data by Chang et al. Just as Bigtable leverages the distributed
+    data storage provided by the Google File System, HBase provides
+    Bigtable-like capabilities on top of Apache Hadoop.
+    """
+
+    homepage = "https://github.com/apache/hbase"
+    url      = "https://github.com/apache/hbase/archive/rel/1.4.12.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/hbase/package.py]:
```suggestion
    url      = "https://github.com/apache/hbase/archive/rel/2.2.2.tar.gz"
```

File: var/spack/repos/builtin/packages/hbase/package.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+    version('2.2.2RC0', sha256='fa78e642225ee13c5b5f9e53f7eb3ec67c0188a776f5525d4d72362a40fcf653')
+    version('2.2.2',    sha256='e9a58946e9adff1cac23a0b261ecf32da32f8d2ced0706af1d04e8a67d582926', preferred=True)
+    version('2.1.8RC0', sha256='e2de965f2dcea83cbd96159d40c2ba9e4a9d0f459d70bc133408886954df627c')
+    version('2.1.8',    sha256='121cea4c554879c8401f676c8eb49e39bd35d41c358e919379ad4a318844c8de')
+
+    depends_on('jdk@1.7:', type='run')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/hbase/package.py]:
```suggestion
    depends_on('java@7:', type='run')
```
There are many Java implementations aside from JDK

File: var/spack/repos/builtin/packages/hbase/package.py (hunk 1 above)
Comment (darmac) [reply var/spack/repos/builtin/packages/hbase/package.py]: fix this in 
https://github.com/spack/spack/pull/14161/commits/ba9de2ffd5f9afe1307b3e3be23414efe779b6d9

File: var/spack/repos/builtin/packages/hbase/package.py (hunk 2 above)
Comment (darmac) [reply var/spack/repos/builtin/packages/hbase/package.py]: fix this in 
https://github.com/spack/spack/pull/14161/commits/ba9de2ffd5f9afe1307b3e3be23414efe779b6d9

File: var/spack/repos/builtin/packages/hbase/package.py (hunk 3 above)
Comment (darmac) [reply var/spack/repos/builtin/packages/hbase/package.py]: fix this in 
https://github.com/spack/spack/pull/14161/commits/ba9de2ffd5f9afe1307b3e3be23414efe779b6d9

**Comment by adamjstewart** [comment]
Closing and reopening to try to fix Travis checks

**Pull request merged**